lib/perl/Genome/Qc/Tool/TestHelpers.t.d/reference.fasta

```diff
@@ -0,0 +1 @@
+/gscmnt/ams1102/info/model_data/2869585698/build106942997/Y.fa
```

Comment:
All the data is paired down to the Y chromosome for faster runtime of the tests. This is the matching reference build to the test bam file I'm using. Is there an alternative that I should rather use?